winterfell/utils/httpResponses.js

```diff
@@ -0,0 +1,33 @@
+function httpResponses(app) {
+    this.app = app;
+};
+
+module.exports = httpResponses;
+
+// Http Codes (wrapper around standard http response status codes)
```

Comment:
Can we just use https://www.npmjs.com/package/http-status-codes instead of rolling this ourselves?